.github/workflows/_charm-build.yaml

```diff
@@ -27,7 +27,7 @@ on:
         value: ${{ jobs.build.outputs.artifacts }}
     secrets:
       CHARMCRAFT_CREDENTIALS:
-        required: true
+        required: false
```

Comment:
given it's actually not used might be able to drop it

Reply:
https://github.com/canonical/identity-team/issues/24